src/routes/Things.js

```diff
@@ -47,8 +49,54 @@ let router = express.Router();
  * }
  **/
 
-router.get('/Things', (req, res) => {
-  res.status(200).send();
+const resource = 'Things';
+
+router.get('/', (req, res) => {
+  db().then((dbInstance) => {
+    dbInstance.Things.findAll().then(things => {
```

Comment:
Is probably ok for createdAt and updatedAt, but we need the id. The filtering is being done in the response because we want to filter what to export, not what to retrieve.